### PR TITLE
26 report create kondratiuk

### PR DESCRIPTION
**Pull request by andrii-kondratiuk**

### CODE REVIEW

File: controllers/reports-controller.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+exports.postReport = async (req, res) => {
+  try {
+    const { placeId, report, userJwt } = req.body;
+    const place = await placeModel.places.findOne({ _id: placeId });
+    if (!place) {
+      throw 'Sorry invalid id of place, try later';
+    }
+
+    const userId = await tokenService.verify(userJwt);
+    console.log(userId);
```
Comment (Dmitry528) [inline controllers/reports-controller.js]: Delete console)

Comment (andrii-kondratiuk) [reply controllers/reports-controller.js]: Done

Comment (andrii-kondratiuk) [reply controllers/reports-controller.js]: Done

**Review by Dmitry528 — APPROVED** [review]
Tested, good work!

**Review by marrybbb — APPROVED** [review]

**Review by Viktor-Hordynskyi — APPROVED** [review]

**Pull request merged**